bin/periphery/src/stats.rs

```diff
@@ -57,6 +80,7 @@ pub struct StatsClient {
   // the handles used to get the stats
   system: sysinfo::System,
   disks: sysinfo::Disks,
+  network: sysinfo::Networks,
```

Comment:
rename field to `networks`, to match `disks` (and the `sysinfo::Networks` struct)

Reply:
Done